diff --git a/dist/commonjs/index.d.ts b/dist/commonjs/index.d.ts
index 1d86e5ec52083b00be390eefad66a03f54c1bffa..8d4bc13621590d805a7dfa978e8cf008a00e7330 100644
--- a/dist/commonjs/index.d.ts
+++ b/dist/commonjs/index.d.ts
@@ -9,4 +9,5 @@ export type BraceExpansionOptions = {
     maxRewrites?: number;
 };
 export declare function expand(str: string, options?: BraceExpansionOptions): string[];
+export default expand;
 //# sourceMappingURL=index.d.ts.map
\ No newline at end of file
diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index 48cf0d3dabc885249c65909c4912712c834786f5..351788b212bc24a559ea16c6e57e11121fd31693 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -330,4 +330,5 @@ function expand_(str, max, maxLength, maxDepth, depth, maxRewrites, isTop) {
     }
     return acc;
 }
+module.exports = Object.assign(expand, exports);
 //# sourceMappingURL=index.js.map
\ No newline at end of file
diff --git a/dist/esm/index.d.ts b/dist/esm/index.d.ts
index 1d86e5ec52083b00be390eefad66a03f54c1bffa..8d4bc13621590d805a7dfa978e8cf008a00e7330 100644
--- a/dist/esm/index.d.ts
+++ b/dist/esm/index.d.ts
@@ -9,4 +9,5 @@ export type BraceExpansionOptions = {
     maxRewrites?: number;
 };
 export declare function expand(str: string, options?: BraceExpansionOptions): string[];
+export default expand;
 //# sourceMappingURL=index.d.ts.map
\ No newline at end of file
diff --git a/dist/esm/index.js b/dist/esm/index.js
index 0e0cc962307eb697dc8139ef4c1f86b82380e5cc..dbce6953f0abf8266d3380ca5382abed2531f9fb 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -326,4 +326,5 @@ function expand_(str, max, maxLength, maxDepth, depth, maxRewrites, isTop) {
     }
     return acc;
 }
+export default expand;
 //# sourceMappingURL=index.js.map
